### ISSUE TITLE
Attempting Werewolf Quest Both Crashing to Desktop and Non-Responsive Locking Up

### ISSUE THREAD

**Issue by CercerDev** [ForgottenGlory/POTD#46]
**Crash Logs**
[Crash_2022_1_22_19-35-19.txt](https://github.com/ForgottenGlory/POTD/files/7919354/Crash_2022_1_22_19-35-19.txt)
[Crash_2022_1_22_20-30-56.txt](https://github.com/ForgottenGlory/POTD/files/7919355/Crash_2022_1_22_20-30-56.txt)
[Crash_2022_1_22_22-56-46.txt](https://github.com/ForgottenGlory/POTD/files/7919436/Crash_2022_1_22_22-56-46.txt)

**Save File**
The save is in the Underforge directly before ingesting the Werewolf blood.
[WerewolfCrashSaves.zip](https://github.com/ForgottenGlory/POTD/files/7919359/WerewolfCrashSaves.zip)

**LS Version**
Path of the Dovahkiin Version: "1.1.0"

**Bug Description**
During the quest "The Silver Hand", after first transforming into a werewolf I am experiencing mostly complete game freezes (video freeze, audio continue loops, game window will not give up top layer priority even to task manager in focus) and the occasional instant CTD. 

**To Reproduce**
Attempt to complete the quest "The Silver Hand",
The problems seem to occur at 2/3 specific points:
-Right after initially gaining control of werewolf form/beginning to sprint the game will often freeze or CTD.
-When Aela lights her torch and begins to talk to the player; the game will run fine for a moment upon loading in and waking up post Whiterun escape but once Aela's conversation "locks in"  it will freeze up/CTD.

**Expected behavior**
Complete the quest by drinking the werewolf blood, running through Whiterun in werewolf form, waking up with Aela by a Silver Hand base and then clearing it out.

**Screenshots**
![Skyrim_Werewolf_Crash_1](https://user-images.githubusercontent.com/98238161/150658311-a2cacd88-ceeb-473c-b9a5-f1f61654ce94.jpg)
![Skyrim_Werewolf_Crash_2](https://user-images.githubusercontent.com/98238161/150658312-1710c44c-6b72-4c88-86de-1872cc851981.jpg)

**Videos**
First video showing the problematic section with an overview of my skills and current gear, game actually didn't crash at all this time.
Loading times are slower than usual due to simultaneously recording to the same drive as it is loading from.
[https://www.youtube.com/watch?v=Ck0yv9cThVM](url)
Second video taken directly after first, game crashed to desktop at video endpoint.
[https://www.youtube.com/watch?v=iDmrDkKF7Jk](url)

**Additional context**
I am level 31 and have one point each in the first 2 unarmed skills, which may be relevant given the 'hand to hand rebalanced' mod shows up in both crash logs, though I have also found similar crash reports from SkyrimVR users relating to the werewolf skeleton in XP32MSSE 4.80.

**Comment by CercerDev** [comment]
I'm not sure if this deserves a separate ticket, if so just say and I'll submit it as such, but as it's also werewolf related it might be fine here:
I ended up having an involuntary werewolf transformation because of Manbeast, and while I didn't crash I also didn't revert back to human, when the time came I got the particle effects and status icons for transforming but nothing actually happened, after which I couldn't attack except by pressing the attack and block buttons at the same time. This could just be a Manbeast bug though so I have also reported it on its nexus page.

I ended up manually transforming and waiting it out to avoid the auto-transform which worked but I shortly later after dying found that the two saves I made after this forced transformation would CTD when trying to load.

Crash log from trying to load the save:
[Crash_2022_1_24_15-14-41.txt](https://github.com/ForgottenGlory/POTD/files/7926629/Crash_2022_1_24_15-14-41.txt)
